draft-ietf-quic-multipath.md

```diff
@@ -192,10 +187,7 @@ Identifier, is that the Path Identifier is used in multipath control
 frames to identify a path, and the Packet Number Space Identifier is
 used in 1-RTT packets and ACK_MP frames to distinguish packet number
 spaces for different paths. Both identifiers have the same value, which
-is the sequence number of the connection ID, if a non-zero connection ID
-is used. If the connection ID is zero length, the Packet Number Space
-Identifier is 0, while the Path Identifier is selected on path
-establishment.
+is the sequence number of the connection ID, when a new path is established.
```

Comment:
Maybe it can be addressed later, but given that the Path Identifier and the PNS ID are now always the same, we may simplify this section and only introduce one concept (i.e., the Path Identifier).

Reply:
Yes, I was planning to raise a separate issue and PR after this one is merged. I think that would be cleaner.